storagenode/console/service.go

```diff
@@ -29,27 +31,22 @@ var (
 
 // DB exposes methods for managing SNO dashboard related data.
 type DB interface {
-	// GetSatelliteIDs returns list of satelliteIDs that storagenode has interacted with
-	// at least once
-	GetSatelliteIDs(ctx context.Context, from, to time.Time) (storj.NodeIDList, error)
-	// GetDailyBandwidthUsed returns slice of daily bandwidth usage for provided time range,
-	// sorted in ascending order
-	GetDailyTotalBandwidthUsed(ctx context.Context, from, to time.Time) ([]BandwidthUsed, error)
-	// GetDailyBandwidthUsed returns slice of daily bandwidth usage for provided time range,
-	// sorted in ascending order for particular satellite
-	GetDailyBandwidthUsed(ctx context.Context, satelliteID storj.NodeID, from, to time.Time) ([]BandwidthUsed, error)
+	// Bandwidth is a getter for Bandwidth db
+	Bandwidth() Bandwidth
```

Comment:
Why not supply `Bandwidth` directly it seems the extra indirection isn't doing much here anymore.